topics/client-basic-auth.md

```diff
@@ -16,39 +16,45 @@ The Basic [authentication scheme](client-auth.md) can be used for logging in use
 
 The basic authentication flow looks as follows:
 
-1. A client makes a request without the `Authorization` header to a specific resource in a server application.
-2. A server responds to a client with a `401` (Unauthorized) response status and uses a `WWW-Authenticate` response header to provide information that the basic authentication scheme is used to protect a route. A typical `WWW-Authenticate` header looks like this:
+1. A client makes a request to a protected resource in a server application without the `Authorization` header.
+2. The server responds with a `401 Unauthorized` response status and uses a `WWW-Authenticate` response header to
+   indicate that Basic authentication is required. A typical `WWW-Authenticate` header looks like this:
 
    ```
    WWW-Authenticate: Basic realm="Access to the '/' path", charset="UTF-8"
    ```
    {style="block"}
 
-   The Ktor client allows you to send credentials without waiting the `WWW-Authenticate` header using the `sendWithoutRequest` [function](#configure).
+   The Ktor client allows you to send credentials preemptively – without waiting for the `WWW-Authenticate` header –
+   by using the [`sendWithoutRequest()` function](#configure).
 
-3. Usually, a client displays a login dialog where a user can enter credentials. Then, a client makes a request with the `Authorization` header containing a username and password pair encoded using Base64, for example:
+3. The client typically prompts the user for credentials. It then makes a request with the `Authorization` header 
+   containing a username and password pair encoded using Base64, for example:
 
    ```
    Authorization: Basic amV0YnJhaW5zOmZvb2Jhcg
    ```
    {style="block"}
 
-4. A server validates credentials sent by the client and responds with the requested content.
+4. The server validates the credentials sent by the client and responds with the requested content.
 
 
 ## Configure basic authentication {id="configure"}
```

Comment:
In the previous file, the headings were in ing-form. Here are imperative, closer to an instruction, which is preferred because it is action-oriented. For consistency, consider changing the ing-form verbs in headings.